tools/bridge/bridge/validator_balance_watcher.py

```diff
@@ -3,17 +3,25 @@
 import gevent
 from eth_utils import from_wei
 
+from bridge.control_events import BalanceCheck
+
 logger = logging.getLogger(__name__)
 
 
 class ValidatorBalanceWatcher:
     def __init__(
-        self, w3, validator_address, poll_interval, balance_warn_threshold
+        self,
+        w3,
+        validator_address,
+        poll_interval,
+        balance_warn_threshold,
+        control_queue,
     ) -> None:
         self.w3 = w3
         self.validator_address = validator_address
         self.poll_interval = poll_interval
         self.balance_warn_threshold = balance_warn_threshold
```

Comment:
Please get rid of the `balance_warn_threshold` field. Just get the value and put it on the queue.

Reply:
Also remove the warning below.